react-native-yoti-doc-scan.podspec

```diff
@@ -2,21 +2,20 @@ require "json"
 
 package = JSON.parse(File.read(File.join(__dir__, "package.json")))
 
-Pod::Spec.new do |s|
-  s.name           = 'react-native-yoti-doc-scan'
-  s.version        = package['version']
-  s.summary        = package['description']
-  s.description    = package['description']
-  s.license        = package['license']
-  s.author         = package['author']
-  s.homepage       = 'https://github.com/getyoti/yoti-doc-scan-react-native'
-  s.source         = { :git => "https://github.com/getyoti/yoti-doc-scan-react-native.git", :tag => "#{s.version}" }
-  s.source_files   = 'ios/**/*.{h,m}'
-  s.platform       = :ios, '11.4'
-  s.dependency 'React'
-  s.dependency 'YotiSDKDocument','2.1.1'
-  s.dependency 'YotiSDKZoom','2.1.1'
-  s.pod_target_xcconfig = {
-    'CLANG_ENABLE_MODULES' => 'NO'
-  }
+Pod::Spec.new do |spec|
+  spec.name                   = "react-native-yoti-doc-scan"
+  spec.version                = package["version"]
+  spec.summary                = package["description"]
+  spec.description            = package["description"]
+  spec.license                = package["license"]
+  spec.author                 = package["author"]
+  spec.homepage               = "https://github.com/getyoti/yoti-doc-scan-react-native"
+  spec.source                 = { :git => "https://github.com/getyoti/yoti-doc-scan-react-native.git", :tag => "#{spec.version}" }
+  spec.source_files           = "ios/**/*.{h,m}""
+  spec.platform               = :ios, "12.0"
```

Comment:
A doubt about this (good to know it if I need to update podspec file next release): Why it cannot be 11.0? I've seen that you added `spec.ios.deployment_target  = "11.0"`, but not sure what's the difference and why now both are needed :happy:

Reply:
`platform` refers to the min. version of Xcode, while `deployment_target` refers to the min. version of iOS.
If only `platform` is set, then CocoaPods will apply that min. version to `deployment_target` as well.

To answer your specific question on why `platform` can't be 11 or 11.4 as it was previously set to: iOS v2.4.0 simply won't work on those versions of Xcode given their associated versions of the Swift compiler. iOS v2.4.0 will thus only work with the Swift compiler associated to Xcode 12. And in order for the `deployment_target` to then not be set to 12 as well, we need to add this to the spec as 11.

Reply:
Oh! I see! Thanks for the explanation :)

Reply:
@mariasenosiain This proved to not work as how I thought. When verifying, I noticed that the target set relative to `platform` is in fact the `deployment_target`, so I've modified the `podspec` and `podfile` to reflect that (I've also adjusted it on our iOS repository, so no issues there).